packages/integrations/image/components/index.ts

```diff
@@ -1,16 +1,57 @@
 /// <reference types="astro/astro-jsx" />
 export { default as Image } from './Image.astro';
 export { default as Picture } from './Picture.astro';
+import type { HTMLAttributes as AllHTMLAttributes } from '../../../astro/types';
```

Comment:
This import won't work when deployed
```suggestion
import type { HTMLAttributes as AllHTMLAttributes } from 'astro/types';
```